test/integration/consul-container/libs/assert/grpc.go

```diff
@@ -26,7 +26,7 @@ func GRPCPing(t *testing.T, addr string) {
 	var msg *fgrpc.PingMessage
 	retries := 0
 	retry.RunWith(&retry.Timer{Timeout: time.Minute, Wait: 25 * time.Millisecond}, t, func(r *retry.R) {
-		t.Logf("making grpc call to %s", addr)
+		r.Logf("making grpc call to %s", addr)
```

Comment:
Do people like `r.Log`? It looks like the only difference is that it dedupes output, but to do so, it buffers all output until it its finished. So if you're running locally, it looks like its hung.